interface/wx/utils.h

```diff
@@ -856,9 +856,8 @@ wxString wxGetUserHome(const wxString& user = wxEmptyString);
 /**
     This function returns the "user id" also known as "login name" under Unix
     (i.e. something like "jsmith"). It uniquely identifies the current user (on
-    this system).  Under Windows or NT, this function first looks in the
-    environment variables USER and LOGNAME; if neither of these is found, the
-    entry @b UserId in the @b wxWidgets section of the WIN.INI file is tried.
+    this system).  Under Windows, this function looks in the
+    environment variables USER and LOGNAME.
```

Comment:
Looking at the code, it seems this is not true:
https://github.com/wxWidgets/wxWidgets/blob/ac3cea5bcd5a3fc3d52648efb4dceacbcc9d0698/src/msw/utils.cpp#L243-L259

Reply:
Fixed to say just "USERNAME" now.